src/pages/congratulations/congratulations.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { withRouter } from 'react-router-dom';
 import { noop, Grid, Button } from 'patternfly-react';
 import { connect, reduxActions } from '../../redux';
+import PfMasthead from '../../components/masthead/masthead';
```

Comment:
Shouldn't this be Masthead?

Reply:
A lot of the test failures are in code I didn't touch, so I'm in the process of figuring out how to fix some of them. Wondering how the code was ever merged before since it's always had these issues. But I will resolve.